drivers/entropy/CMakeLists.txt

```diff
@@ -1,10 +1,8 @@
-zephyr_library()
```

Comment:
This is not nice. Is there another way of doing this? Otherwise CMake will complain about an empty library.

Reply:
For now I saw 2 ways to avoid the empty library issue:
1) Remove `zephyr_library()` and use `zephyr_sources_ifdef()` as it done here;
2) Modify extensions.cmake in Zephyr to make out-of-tree drivers look like they are in the Zehpyr's tree;

Maybe the 3rd way is to do a trick with Kconfig by using another way to include driver/entropy folder even if CONFIG_ENTROPY_HAS_DRIVER is enabled.